src/builder.ts

```diff
@@ -286,6 +289,7 @@ export type TypeToWalk =
   | { type: 'named'; value: GraphQLNamedType }
   | { type: 'input'; value: NexusShapedInput }
   | { type: 'object'; value: NexusShapedOutput }
+  | { type: 'subscription'; value: NexusSubscriptionTypeConfig }
```

Comment:
Given that subscription is just an object type, what's the reason for needing an additional type to walk here? It looks like it's not even walking the type in a special way. Can't we just type the surface API but then return a simple `ObjectTypeDef` ?